### DISCUSSION TITLE
Sort by whether is ignored by git?

### DISCUSSION THREAD

**Discussion by dupeiran001** [nvim-tree/nvim-tree.lua#2614] [Q&A]
I'd like to show all files that hasn't been ignored by git first, so the custom sorter may help. However the [custom sorter](https://github.com/nvim-tree/nvim-tree.lua/blob/f1b3e6a7eb92da492bd693257367d9256839ed3d/doc/nvim-tree-lua.txt#L667-L690) doesn't has an field of whether it's is ignored. So is it possible to be implemented with API?

**Comment by alex-courtis** [comment]
We can do that: more data can be added i.e. `GitStatus` as per https://github.com/nvim-tree/nvim-tree.lua/blob/96a783fbd606a458bcce2ef8041240a8b94510ce/lua/nvim-tree/node.lua

Please raise a feature request to add missing data. As usual, [Pull Requests](https://github.com/nvim-tree/nvim-tree.lua/blob/master/CONTRIBUTING.md) are most gratefully appreciated.
